apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -88,6 +110,24 @@ const buildSpacingExample = token => {
     return `<div class="c-tokensTable-example-spacing-container"><div class="c-tokensTable-example--spacing" style="${cssVariable}";></div></div>`;
 };
 
+/**
+* Builds an example font/typography element to show on the token list item
+* @param {string} token - the token value i.e. 
+* @returns {string} - the typography example HTML string
+*/
+const buildFontExample = token => { 
+    const {
+        fontSize, lineHeight, fontWeight, textDecoration 
+    } = splitFontAliasToken(token);
+    const cssVariables = [
+   `--example-font-size: ${fontSize}px`,
+   `--example-font-line-height: ${lineHeight}px`,
+   `--example-font-weight: ${fontWeight}`,
+   textDecoration && `--example-font-text-decoration: ${textDecoration}`
+    ].filter(Boolean);
```

Comment:
What does this filter do? Looks like it might filter out the text decoration if it isn't defined?

Reply:
yep, the filter is needed to filter undefined/falsy values from the array as not all examples/tokens need/have `text-decoration styles. 

`passing filter(Boolean)` is a neat trick to do so. 

https://michaeluloth.com/filter-boolean

Reply:
I think we would typically do something like this to avoid the need to filter, but your way is probably nicer here:
```
const cssVariables = [
    `--example-font-size: ${fontSize}px`,
    `--example-font-line-height: ${lineHeight}px`,
    `--example-font-weight: ${fontWeight}`,
    ...(textDecoration ? [`--example-font-text-decoration: ${textDecoration}`] : [])
];
```